### PR TITLE
[Doc Feature][People Picker]

### PR DESCRIPTION
**Pull request by v-rpatkur**

### CODE REVIEW

File: msteams-platform/task-modules-and-cards/cards-and-task-modules.md [hunk 1]
```diff
@@ -25,12 +25,13 @@ The following types of cards are supported in bots for Teams:
...
 You can format card text using a subset of XML or HTML formatting or Markdown depending on the card type.
 
+People Picker in Adaptive Card helps to search, select, reassign, and preselect users within chat or channel.
+
 You can add and respond to card actions that:
 * Open a URL
 * Send messages and payload to the bot
 * Initiate OAuth flow
 
-You can give a [dynamic search](~/task-modules-and-cards/cards/dynamic-search.md) experience within a large dataset using type ahead control in Adaptive Cards. You can also do a type-ahead static search within limited number of choices. You can invoke the task modules in channel or personal tabs, bots, or deep links. Your user's experience for any workflows that requires data input can be improved by adding a task module to the user's tab. You can invoke task modules from Teams bots using buttons on Adaptive Cards and Bot Framework cards.
```
Comment (sowrabh-msft) [inline msteams-platform/task-modules-and-cards/cards-and-task-modules.md:33]: Is removing this intentional?

**Review by sowrabh-msft — DISMISSED** [review]

File: msteams-platform/task-modules-and-cards/cards/people-picker.md [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+
+You can use `Action.Submit` or `Action.Execute` to submit selected data to your bot. The `invoke` payload received on your bot is a list of AAD IDs or the IDs provided in static list.
+In People Picker, when a user is selected in the control, the `AAD ID` of the user is the value sent back. The `AAD ID` is a string and uniquely identifies a user in the directory.
+
+The format of the value submitted to the bot depends on the value of the `isMultiSelect` property:
+
+|value of `isMultiSelect`|Format|
+|--|--|
+|false _(single select)_|<selected_AAD_ID>.|
+|true _(multi select)_|<selected_AAD_ID_1>,<selected_AAD_ID_2>,<selected_AAD_ID_3>.|
```
Comment (karthikbaskar) [inline msteams-platform/task-modules-and-cards/cards/people-picker.md:163]: Please remove the period after <selected_AAD_ID_3>

this is how it should be:
|true _(multi select)_|<selected_AAD_ID_1>,<selected_AAD_ID_2>,<selected_AAD_ID_3>|

**Review by v-abirade — DISMISSED** [review]
Approved

**Review by v-abirade — DISMISSED** [review]
I will make the suggested changes and update the PR.

File: msteams-platform/task-modules-and-cards/cards/people-picker.md [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+#### dataset
+The following table provides predefined values as **dataset** for people picker:   
+
+|dataset|Search Scope
+|--|--|
+|**graph.microsoft.com/users** |Search all members across the organization.|
+|**graph.microsoft.com/users?scope=currentContext** |Search within the members of the current conversation, such as chat or channel in which the particular card is sent.|      
+
+> [!NOTE]
+> Currently, the search functionality for all the members across the organization is supported in 1:1 chats with bots, Adaptive Card task modules and tabs only. It is not supported in other 1:1 chats, group chats or channels.
```
[the remark this answers is not in the document]
Comment (shiladityasaha) [reply msteams-platform/task-modules-and-cards/cards/people-picker.md:110]: @v-abirade can you remove this note?

File: msteams-platform/task-modules-and-cards/cards/people-picker.md (hunk 2 above)
Comment (shiladityasaha) [reply msteams-platform/task-modules-and-cards/cards/people-picker.md:163]: @v-abirade can you make this change?

**Review by shiladityasaha — DISMISSED** [review]
@v-abirade can you make the changes suggested by @sowrabh-msft and @karthikbaskar before publishing?

File: msteams-platform/task-modules-and-cards/cards-and-task-modules.md [hunk 4]
```diff
@@ -25,12 +25,13 @@ The following types of cards are supported in bots for Teams:
 
 You can format card text using a subset of XML or HTML formatting or Markdown depending on the card type.
 
+People Picker in Adaptive Card helps to search, select, reassign, and preselect users within chat or channel.
```
[the remark this answers is not in the document]
Comment (v-abirade) [reply msteams-platform/task-modules-and-cards/cards-and-task-modules.md:28]: Done

File: msteams-platform/task-modules-and-cards/cards/people-picker.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (v-abirade) [reply msteams-platform/task-modules-and-cards/cards/people-picker.md:110]: Removed

File: msteams-platform/task-modules-and-cards/cards-and-task-modules.md (hunk 1 above)
Comment (v-abirade) [reply msteams-platform/task-modules-and-cards/cards-and-task-modules.md:33]: Added

File: msteams-platform/task-modules-and-cards/cards/people-picker.md (hunk 2 above)
Comment (v-abirade) [reply msteams-platform/task-modules-and-cards/cards/people-picker.md:163]: removed

File: msteams-platform/task-modules-and-cards/cards-and-task-modules.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (sowrabh-msft) [reply msteams-platform/task-modules-and-cards/cards-and-task-modules.md]: Looks like it got missed. Please add link

**Review by sowrabh-msft — DISMISSED** [review]

**Review by v-abirade — APPROVED** [review]
Approved

**Review by surbhigupta12 — APPROVED** [review]
Approved

**Pull request merged**